public/app/core/services/analytics.ts

```diff
@@ -7,14 +7,19 @@ export class Analytics {
   constructor(private $rootScope, private $location) {}
 
   gaInit() {
-    $.getScript('https://www.google-analytics.com/analytics.js'); // jQuery shortcut
+    $.ajax({
+      url: 'https://www.google-analytics.com/analytics.js',
+      dataType: 'script',
+      cache: true,
+    });
     var ga = ((<any>window).ga =
       (<any>window).ga ||
       function() {
         (ga.q = ga.q || []).push(arguments);
       });
     ga.l = +new Date();
     ga('create', (<any>config).googleAnalyticsId, 'auto');
+    ga('set', 'anonymizeIp', true);
```

Comment:
I think this should be a config option. Some users may have a users authorisation to collect their full IP or their users are not in EU. Maybe also GA cookie lifetime should be configurable, because GDPR.